tests/test_gpg.py

```diff
@@ -655,31 +655,31 @@ def tearDownClass(self):
   def test_export_pubkey(self):
     """ export a public key and make sure the parameters are the right ones:
 
-      since there's very little we can do to check rsa key parameters are right
-      we pre-exported the public key to an ssh key, which we can load with
-      cryptography for the sake of comparison """
+      since there's very little we can do to check key parameters are right
+      we pre-exported the public key to an x.509 SubjectPublicKeyInfo key,
+      which we can load with cryptography for the sake of comparison """
 
     # export our gpg key, using our functions
     key_data = export_pubkey(self.default_keyid, homedir=self.gnupg_home)
     our_exported_key = dsa_create_pubkey(key_data)
 
     # load the equivalent ssh key, and make sure that we get the same RSA key
```

Comment:
Nit: this is not an ssh key any more, right?

Reply:
and not an rsa key either. 🤦  thanks for catching!

Reply:
Updated and squashed into 492f7e5